### DISCUSSION TITLE
"Replace" playlist doesn't work

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#124] [Fixed Issues] [answered]
**rAudio 1 • 20210115**

In library view: "Replace" playlist doesn't work.
("Replace + Play" is o.k.)

**Comment by ceejayemm** [comment, marked as answer]
After update 20210120, seems OK to me (RPi 3B+, Merus Audio Amp).

Chris

**Comment by klassikmann** [comment]
Confirmed. Solved in "rAudio 1 • 20210120".
